README.md

```diff
@@ -8,12 +8,9 @@
   <a title="Learn more about Copy Word in Cursor" href="http://github.com/alefragnani/vscode-copy-word"><img src="https://raw.githubusercontent.com/alefragnani/vscode-copy-word/master/images/vscode-copy-word-logo-readme.png" alt="Copy Word in Cursor Logo" width="70%" /></a>
 </p>
 
-# What's new in Copy Word in Cursor 3.6
+# What's new in Copy Word in Cursor 3.7
```

Comment:
Thanks, but undo file changes. Whenever I update the extension, I like to review its content.